replate/src/utils/axiosWithAuth.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+// Build a module that "creates" a new "instance" of the axios object
+// - baseURL
+// - headers object -> authorization header with the token
+
+window.axios = axios;
```

Comment:
why are you setting axios to the window object?  is that so you don' t have to import axios in other files?